src/assets/DashboardContent.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchRecent, fetchPostByRoom, fetchRooms } from '../store/actions';
+import { Layout } from 'antd';
+
+// import Feed from './Feed';
```

Comment:
Commented out code, saw this removed from another file so I'm guessing it's not needed?

Reply:
good catch